### DISCUSSION TITLE
Webconsole Password reset

### DISCUSSION THREAD

**Discussion by nico-hl** [rustdesk/rustdesk#4977] [General]
Hey there, I hope someone can help me 
I installed a rustdesk selfhost server on ubuntu. everything works fine. but today I couldn't login to the web console because my password is wrong. I don't know why but I have a few clients out there with rust desk and my public id and I don't want to touch every client again. Is there a way that I can change or reset the password for the webconsole (I mean that section where I can get the QR codes and my RustDesk installer files)? 

nico

**Comment by dinger1986** [comment]
Presumably you mean the console where you download the configs and you installed with techaholds install script? 

If so this is the wrong place to ask it should be on their GitHub but if you do `cat /etc/systemd/system/gohttpserver.service` and look at the end of the `ExecStart` line you will find the password after `admin:`

**Comment by nico-hl** [comment]
Yes, you are my hero :) Thank you sooo much

**Comment by RickkeeC** [comment]
Hesus J Christie, they/them made it so easy with the script but failed to mention this little detail.  Cost me 3 hours of grief until I finally found this post! Bravo

**Comment by rustdesk** [comment]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/faq/#after-changing-the-admin-password-in-the-web-console-i-cannot-log-in-is-there-a-simple-way-to-reset-the-password
